plugins/sdm/accessbot.py

```diff
@@ -119,6 +130,18 @@ def show_roles(self, message, match):
             return
         yield from self.get_show_roles_helper().execute(message)
 
+    @re_botcmd(pattern=r'.+', flags=re.IGNORECASE, prefixed=False)
+    def match_alias(self, message, _):
```

Comment:
This function should call a method in a helper class, similar to what we do in `show_resources` or any other functionality